infrastructure/openhim-base-config.json

```diff
@@ -1524,6 +1524,57 @@
       "priority": 1,
       "name": "New Death InProgress"
     },
+    {
+      "methods": ["POST"],
+      "type": "http",
+      "allow": [],
+      "whitelist": [],
+      "authType": "public",
+      "matchContentTypes": [],
+      "properties": [],
+      "txViewAcl": [],
+      "txViewFullAcl": [],
+      "txRerunAcl": [],
+      "status": "enabled",
+      "rewriteUrls": false,
+      "addAutoRewriteRules": true,
+      "autoRetryEnabled": false,
+      "autoRetryPeriodMinutes": 60,
+      "routes": [
+        {
+          "type": "http",
+          "status": "enabled",
+          "forwardAuthHeader": true,
+          "name": "Chatbot mediator",
+          "secured": false,
+          "host": "ocrvs-chatbot-mediator",
+          "port": 8050,
+          "path": "",
+          "pathTransform": "",
+          "primary": true,
+          "username": "",
+          "password": ""
+        }
+      ],
+      "requestBody": true,
+      "responseBody": true,
+      "rewriteUrlsConfig": [],
+      "name": "Chatbot mediator",
+      "urlPattern": "^/search",
```

Comment:
Should we try to give a more specific url pattern for chatbot search feature ? something like: ` ^/chat-bot/search`

Reply:
I think that we should.  But I actually think we should have a dedicated endpoint for chatbot search that doesn't return all the data points that the search endpoint does for security reasons.  Right now though, this is a prototype that is useless without the chatbot mediator and chatbot application.  So for now I think this is fine.